Dockerfile

```
@@ -0,0 +1,42 @@
+# Use a base image with Java and MySQL installed
+FROM ubuntu:latest
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)